packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.html

```diff
@@ -26,6 +26,41 @@
                                       [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
                     </nui-chart-marker>
                 </nui-rich-legend-tile>
+
+                <div description class="d-flex align-items-center px-2">
+                    <div class="description-container d-flex flex-column justify-content-center">
+                        <div *ngIf="(legendSeries?.link? false : true) else elseLinkHref">
+                            <div class= "nui-text-link-small"
+                                (click)="onInteraction(legendSeries)">
+                                <div class="description description-primary" 
+                                    [title]="legendSeries.legendDescriptionPrimary">
+                                        {{legendSeries.legendDescriptionPrimary}}
+                                </div>
+                                <div class="description description-secondary" 
+                                    [title]="legendSeries.legendDescriptionSecondary">
+                                        {{legendSeries.legendDescriptionSecondary}}
+                                </div>
+                            </div>
+                        </div>
+                        <ng-template #elseLinkHref>
+                            <div (click)="$event.stopPropagation()">
+                                <a class="nui-text-link-small"
+                                        [href]="legendSeries.link" 
+                                        target="_blank">
```

Comment:
I don't know if we should use target `_blank` here. I think users might appreciate being able to CTRL click or right click to open in a new tab if they want. If they want to open the new page in the same tab, they should be able to just click it. Perhaps this is another UX question for Thursday?

Reply:
That makes complete sense. Removed for now.